Library/Homebrew/cmd/update.sh

```diff
@@ -2,8 +2,11 @@
 #:
 #:  Fetch the newest version of Homebrew and all formulae from GitHub using `git`(1) and perform any necessary migrations.
 #:
-#:       --merge                         `git merge` is used to include updates (rather than `git rebase`).
-#:       --force                         Always do a slower, full update check (even if unnecessary).
+#:          --merge                      `git merge` is used to include updates (rather than `git rebase`).
+#:      -f, --force                      Always do a slower, full update check (even if unnecessary).
+#:      -v, --verbose                    Print the directories checked and `git` operations performed.
+#:      -d, --debug                      Display any debugging information.
```

Comment:
Would be good to elaborate on exactly what this is doing. I believe it's going to output every single command executed in Bash.

Reply:
Sure, can elaborate. I just borrowed that description from `Homebrew::CLI::Parser.global_options`.

Reply:
Thanks!

Reply:
This is now better described in 8b85625.